src/components/faq-page/GeneralFaqPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import FaqSection from '@/components/faq-page/FaqSection';
+import GeneralInfoPage from '@/components/faq-page/GeneralInfoPage';
+import { Banner } from '@/components/ui/banner/Banner';
+import Footer from '@/components/ui/Footer/Footer';
+
+const GeneralFaqPage = () => {
+  return (
+    <div>
+      <Banner
+        image="/images/samahan-members.png"
+        imageAlt="SAMAHAN Members Image"
+        subtitle="FAQ"
+        title="SAMAHAN"
+      />
+      <FaqSection />
+      <GeneralInfoPage />
+      <Footer />
```

Comment:
remove footer as it is already being applied to all pages since it is being used in the layout file

Reply:
also create a folder named `frequently-asked-questions` within the `src/app/` folder. Inside the faq folder, add a page.tsx file that imports the `GeneralFaqPage.tsx`